parm/fv3lam_rrfs.xml

```diff
@@ -3624,5 +3630,11 @@
        <scale>4.0</scale>
        </param>
 
+       <param>
```

Comment:
@JesseMeng-NOAA Can you also update the xml file?

Reply:
Done